### ISSUE TITLE
Terrible performance in some GUIs on Linux

### ISSUE THREAD

**Issue by CyraLX** [modrinth/code#1222]
### Describe the bug

When browsing the Modrinth App (Theseus v0.7.1; `.deb` package) in some guis on Linux the app slows to a crawl, 1 or less fps for sure.
The GUIs include (and likely not limited to):

- Home page
- Browse page (Mods, datapacks, resource packs, ect...)
- Profile `Content` tab
- Profile `Options` tab
- Profile `Logs` tab and this one in particular locks up the app and needs to be killed/restarted

This doesn't effect all GUIs such as the app's Settings page and the Library page

### Steps to reproduce

1. Go to listed GUIs on the Modrinth App while on Linux
2. Issue noticeable while trying to interact with the page (hovering over things, clicking, scrolling, ect...)

### Expected behavior

No performance issues

### System information

App Version: Theseus v0.7.1
Package: `.deb`
OS: Linux x86 64bit (Pop!_OS 22.04 LTS)
GPU Driver: nvidia-driver-550

### Additional context

While this could be an issue on my end and something changed that is a result of this, but previously this wasn't happening, only after a few months of not using linux, returning to it found this app in this state after updating everything

**Comment by TrialDragon** [comment]
I'm experiencing the exact same severe performance issues on Fedora 40. It leaves the app practically unusable.

App Version: Theseus v0.7.1
Package: AppImage
OS: Linux 6.8.11-300.fc40.x86_64 , 64-bit , Fedora 40 Workstation Edition
GPU Driver: nvidia-driver-550 (NVIDIA GeForce RTX™ 2060)

**Comment by MarcPG1905** [comment]
Having the same issues which makes me pretty much unable to use the app without watching a movie or something while waiting for the frame to get rendered. Potentially having 5 spf (not fps). Also the app just crashes when opening logs of an instance and I had to add `WEBKIT_DISABLE_DMABUF_RENDERER=1` to the .Desktop file.

App Version: `Theseus v0.7.1`
Package: Both `AppImage` and `Flatpak`
System: Both `Fedora 39` and `Fedora 40`
GPU: `NVIDIA GeForce RTX 4070`
Driver: `550.90.07`

**Comment by blryface** [comment]
Im pretty sure it's just how tauri works on linux which causes performance to be fucked

**Comment by TrialDragon** [comment]
> Im pretty sure it's just how tauri works on linux which causes performance to be fucked

No. I did some further testing and Theseus v0.6.3 doesn't (and presumably all versions prior to v0.6.3 don't) have these performance issues and the profile logs page doesn't crash the application in that case. This seems to be a new issue introduced in Theseus v0.7.0+ separate of normal Tauri performance issues.

**Comment by monomere** [comment]
Duplicate? #1216
Also possibly related tauri issue: https://github.com/tauri-apps/tauri/issues/6200

**Comment by Norbiros** [comment]
Tauri is working on providing other wen engines like Chromium Embedded Framework as alternative for Webkit2GTK

**Comment by Android789515** [comment]
> Having the same issues which makes me pretty much unable to use the app without watching a movie or something while waiting for the frame to get rendered. Potentially having 5 spf (not fps). Also the app just crashes when opening logs of an instance and I had to add `WEBKIT_DISABLE_DMABUF_RENDERER=1` to the .Desktop file.
> 
> App Version: `Theseus v0.7.1` Package: Both `AppImage` and `Flatpak` System: Both `Fedora 39` and `Fedora 40` GPU: `NVIDIA GeForce RTX 4070` Driver: `550.90.07`

How do I do this?

**Comment by Prospector** [comment]
Consolidated into #3057
